models/JournalEntry.model.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const JournalEntrySchema = new mongoose.Schema({
+    date: {
+        type: String,
+        required: true
+    },  
+});
+
+const JournalEntry = mongoose.model('journalEntry', JournalEntrySchema);
+module.exports = JournalEntry;
```

Comment:
Looks good for initially showing that the schema is working properly. Just need to add the rest of the information for a journal entry.